packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -732,7 +732,7 @@ const DatePickerInput = (props) => {
       size="2"
       pattern="\d*"
       aria-required={required}
-      aria-label={intl.formatMessage({ id: 'Terra.datePicker.dayLabel' })}
+      aria-label={`${ariaLabel || ''} ${intl.formatMessage({ id: 'Terra.datePicker.dayLabel' })}`}
```

Comment:
Since `ariaLabel` is being used in https://github.com/cerner/terra-framework/blob/main/packages/terra-date-picker/src/DateInput.jsx#L853C178-L853C183 . Can you re-validate SR if there's any repeated announcement in both SR

Reply:
Yes Sugan, Have tested this scenario few times. It is not being read repeatedly.